modern/.env

```diff
@@ -1,3 +1,4 @@
 ESLINT_NO_DEV_ERRORS=false
 REACT_APP_URL_NAME='localhost:8082'
 REACT_APP_VERSION=$npm_package_version
+REACT_APP_TOOLTIP_DELAY=750
```

Comment:
No no no. That's not the right way. You have to define override the default in the theme.